### PR TITLE
Directly defining kernel class option

### PR DESCRIPTION
**Pull request by lchrusciel**
Based on #23

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,16 +36,20 @@ To customize your test suite configuration you can add a few more options to php
...
+    <server name="KERNEL_CLASS" value="/path/to/kernel/class" />
     <server name="EXPECTED_RESPONSE_DIR" value="/path/to/expected/responses/" />
     <server name="MOCKED_RESPONSE_DIR" value="/path/to/mocked/responses/" />
     <server name="OPEN_ERROR_IN_BROWSER" value="true/false" />
     <server name="OPEN_BROWSER_COMMAND" value="open %s" />
+    <server name="IS_DOCTRINE_ORM_SUPPORTED" value="true/false" />
 </php>
 ```
  * `KERNEL_DIR` variable contains a path to kernel of your project. If not set, WebTestCase will look for AppKernel in the folder where you have your phpunit.xml file.
+ * `KERNEL_CLASS` allows you to specify exactly which class in which folder should be used in order to set the Kernel up.
```
Comment (michalmarcinkowski) [inline README.md]: in order to **setup** the Kernel

File: README.md [hunk 2]
```diff
@@ -36,16 +36,20 @@ To customize your test suite configuration you can add a few more options to php
 ```xml
 <php>
     <server name="KERNEL_DIR" value="/path/to/dir/with/kernel" />
```
Comment (michalmarcinkowski) [inline README.md]: Is this parameter still used somewhere?

Comment (lchrusciel) [inline README.md]: Yup. It is used by basic WebTestCase.  We will have it as an optional parameter.

Comment (michalmarcinkowski) [inline README.md]: :+1:

Comment (pamil) [inline README.md]: Can we use `dirname(KERNEL_CLASS)` as a fallback?

File: README.md [hunk 3]
```diff
@@ -36,16 +36,20 @@ To customize your test suite configuration you can add a few more options to php
...
+    <server name="KERNEL_CLASS" value="/path/to/kernel/class" />
```
Comment (pamil) [inline README.md]: IMHO it's quite confusing since it's kernel class **path**, not a class name.

File: src/ApiTestCase.php [hunk 4]
```diff
@@ -195,7 +245,7 @@ protected function loadFixturesFrom($source)
         $loader = new Loader();
 
         $baseDirectory = $this->getFixturesFolder();
-        $source = $baseDirectory.'/'.$source;
+        $source = $baseDirectory.DIRECTORY_SEPARATOR.$source;
```
Comment (pamil) [inline src/ApiTestCase.php]: `\DIRECTORY_SEPARATOR` since it's core PHP constant

Comment (lchrusciel) [inline src/ApiTestCase.php]: :+1:

File: README.md (hunk 3 above)
Comment (lchrusciel) [inline README.md]: :+1:

**Pull request merged**